src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let price = 0;
+
+  if (days >= 7) {
+    price = days * 40 - 50;
+  } else if (days >= 3) {
+    price = days * 40 - 20;
+  } else {
```

Comment:
I don't know if this condition is appropriate here or no, maybe it's better to write just 2 if conditions without else if

Reply:
thanks! fixed all mistakes